static/app/components/replays/replayController.tsx

```diff
@@ -30,6 +32,8 @@ import useOrganization from 'sentry/utils/useOrganization';
 
 const SECOND = 1000;
 
+const COMPACT_WIDTH_BREAKPOINT = 500;
```

Comment:
Was `400` before. Seems arbitrary. 

Wider now because we have the bar and start/total time flanking it.